packages/tools/u-boot/package.mk

```diff
@@ -21,6 +21,12 @@ PKG_NEED_UNPACK="$PROJECT_DIR/$PROJECT/bootloader"
 [ -n "$DEVICE" ] && PKG_NEED_UNPACK+=" $PROJECT_DIR/$PROJECT/devices/$DEVICE/bootloader"
 
 case "$PROJECT" in
+  Amlogic)
+    PKG_VERSION="2019.07"
```

Comment:
Just bump the u-boot version

Reply:
could be done in a follow up pr too i guess (as chewitt is currently short on time)

Reply:
do we want experimental (Amlogic) patches applied to all u-boot things? .. or we keep Amlogic separate?

Reply:
The patch looks fine to me to apply globally